### ISSUE TITLE
nastruct: major and minor groove do not match to x3dna

### ISSUE THREAD

**Issue by hainm** [Amber-MD/cpptraj#121]
Here is the data from x3dna

```bash
Minor and major groove widths: direct P-P distances and refined P-P distances
   which take into account the directions of the sugar-phosphate backbones

   (Subtract 5.8 Angstrom from the values to take account of the vdw radii
    of the phosphate groups, and for comparison with FreeHelix and Curves.)

Ref: M. A. El Hassan and C. R. Calladine (1998). ``Two Distinct Modes of
     Protein-induced Bending in DNA.'' J. Mol. Biol., v282, pp331-343.

                  Minor Groove        Major Groove
                 P-P     Refined     P-P     Refined
   1 gG/Cc       ---       ---       ---       ---
   2 GG/CC       ---       ---       ---       ---
   3 GC/GC      15.8       ---      19.9       ---
   4 CG/CG      15.8      14.6      20.8      19.6
   5 GC/GC      15.5       ---      20.0       ---
   6 CC/GG       ---       ---       ---       ---
   7 Cc/gG       ---       ---       ---       ---
****************************************************************************
```

This is from cpptraj
```bash
Major
['1G16C', '2G15C', '3G14C', '4C13G', '5G12C', '6C11G', '7C10G', '8C9G']
[[  0.          18.60012245  18.7782402   20.35052109  19.96086884
   18.45940208  18.78943062   0.        ]]

Minor
['1G16C', '2G15C', '3G14C', '4C13G', '5G12C', '6C11G', '7C10G', '8C9G']
[[ 13.32927036  13.403409    13.57159901  13.25856209  13.41105175
   13.26655865  13.43054485  13.4557209 ]]
```

**Comment by hainm** [comment]
I only perform calculation for single single structure extracted from my tip3p simulations. So I guess the difference in major and minor is due to I do not choose a good reference structure?

**Comment by drroe** [comment]
@hainm A different procedure is currently used to calc groove widths than is used in 3DNA. I'm working on it now.

**Comment by hainm** [comment]
This will be great, bro.

I had used curve+ and x3dna but did not really satisfy. 

X3dna require writing to pdb file first. Curve require change ext to trj.

Right now i prefer to use cpptraj for basic calc like major, minor, step, ...

**Comment by drroe** [comment]
Should be addressed by #152. Specify `groovecalc 3dna` to `nastruct`.

**Comment by hainm** [comment]
and close with https://github.com/Amber-MD/pytraj/pull/984
